### PR TITLE
Stackdriver exporters: set user agent on gRPC channel.

### PR DESCRIPTION
**Pull request by g-easy**

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -37,6 +37,62 @@ http_archive(
     urls = ["https://github.com/google/benchmark/archive/master.zip"],
 )
 
+# gRPC and Prometheus supply different BUILD files for zlib.
```
Comment (g-easy) [inline WORKSPACE]: Ignore this, it's the build fix from #233

**Comment by g-easy** [comment]
Please note that the generated user-agent string (I verified this with the gRPC client+server example) looks like:

```
opencensus-cpp [0.3.0-dev] grpc-c++/1.17.0-dev grpc-c/7.0.0-dev (linux; chttp2; gizmo)
```

Specifically: `opencensus-cpp [0.3.0-dev]` versus `opencensus-cpp/0.3.0-dev` with a slash.

**Comment by g-easy** [comment]
Thanks for the approval! Does the user-agent string look correct though?

**Comment by rakyll** [comment]
Looks correct to me, cc @songy23.

**Comment by rakyll** [comment]
Not sure if space between opencensus-cpp and version is a correct format. User agent is a multi part header where separator is a space. @songy23, any reasons why we left a space between language and version?

**Comment by rakyll** [comment]
We can simply can go with "opencensus-cpp/0.3.0-dev", but need to update the user agent in all other language implementations too.

**Comment by g-easy** [comment]
I think the question was intended to be: how did we end up with the "name [ver]" format in the first place?

**Pull request merged**